include/nbl/video/SPhysicalDeviceLimits.h

```diff
@@ -0,0 +1,497 @@
+
+    /* PushDescriptorPropertiesKHR *//* provided by VK_KHR_push_descriptor */
+    //uint32_t           maxPushDescriptors;
+
+    /* RayTracingPipelinePropertiesKHR *//* provided by VK_KHR_ray_tracing_pipeline */
+    uint32_t           shaderGroupHandleSize;
+    uint32_t           maxRayRecursionDepth;
+    uint32_t           maxShaderGroupStride;
+    uint32_t           shaderGroupBaseAlignment;
+    uint32_t           shaderGroupHandleCaptureReplaySize;
+    uint32_t           maxRayDispatchInvocationCount;
+    uint32_t           shaderGroupHandleAlignment;
+    uint32_t           maxRayHitAttributeSize;
+
+    /* FloatControlsPropertiesKHR *//* VK_KHR_shader_float_controls *//* MOVED TO Vulkan 1.2 Core  */
+    /* ShaderIntegerDotProductFeaturesKHR *//* VK_KHR_shader_integer_dot_product *//* MOVED TO Vulkan 1.3 Core  */
+    /* TimelineSemaphorePropertiesKHR *//* VK_KHR_timeline_semaphore *//* MOVED TO Vulkan 1.2 Core  */
+    /* VK_KHX_multiview *//* replaced by VK_KHR_multiview */
+
+    /* MultiviewPerViewAttributesPropertiesNVX *//* VK_NVX_multiview_per_view_attributes */
+    // VkBool32           perViewPositionAllComponents;
+
+    /* VK_NVX_raytracing *//* Preview Extension of raytracing, useless*/
+    
+    /* CooperativeMatrixPropertiesNV *//* VK_NV_cooperative_matrix */
+    // VkShaderStageFlags    cooperativeMatrixSupportedStages;
+
+    /* DeviceGeneratedCommandsPropertiesNV *//* VK_NV_device_generated_commands */
+    //uint32_t           maxGraphicsShaderGroupCount;
+    //uint32_t           maxIndirectSequenceCount;
+    //uint32_t           maxIndirectCommandsTokenCount;
+    //uint32_t           maxIndirectCommandsStreamCount;
+    //uint32_t           maxIndirectCommandsTokenOffset;
+    //uint32_t           maxIndirectCommandsStreamStride;
+    //uint32_t           minSequencesCountBufferOffsetAlignment;
+    //uint32_t           minSequencesIndexBufferOffsetAlignment;
+    //uint32_t           minIndirectCommandsBufferOffsetAlignment;
+
+    /* FragmentShadingRateEnumsPropertiesNV *//* VK_NV_fragment_shading_rate_enums */
+    // VkSampleCountFlagBits    maxFragmentShadingRateInvocationCount;
+
+    /* MeshShaderPropertiesNV *//* VK_NV_mesh_shader */
+    //uint32_t           maxDrawMeshTasksCount;
+    //uint32_t           maxTaskWorkGroupInvocations;
+    //uint32_t           maxTaskWorkGroupSize[3];
+    //uint32_t           maxTaskTotalMemorySize;
+    //uint32_t           maxTaskOutputCount;
+    //uint32_t           maxMeshWorkGroupInvocations;
+    //uint32_t           maxMeshWorkGroupSize[3];
+    //uint32_t           maxMeshTotalMemorySize;
+    //uint32_t           maxMeshOutputVertices;
+    //uint32_t           maxMeshOutputPrimitives;
+    //uint32_t           maxMeshMultiviewViewCount;
+    //uint32_t           meshOutputPerVertexGranularity;
+    //uint32_t           meshOutputPerPrimitiveGranularity;
+
+    /* VK_NV_ray_tracing *//* useless because of VK_KHR_ray_tracing_pipeline */
```

Comment:
yep not exposing that

Reply:
`KHR_Acceleration_structure` and `KHR_ray_tracing_pipeline` cover them completely